packages-content-model/roosterjs-content-model-plugins/lib/tableEdit/editors/features/TableInserter.ts

```diff
@@ -0,0 +1,191 @@
+    const tableRect = table && viewPort ? getIntersectedRect([table], [viewPort]) : null;
+
+    // set inserter position
+    if (tdRect && tableRect) {
+        const document = td.ownerDocument;
+        const createElementData = getInsertElementData(
+            isHorizontal,
+            editor.isDarkMode(),
+            isRTL,
+            'red' //editor.getDefaultFormat().backgroundColor || 'white'
+            // TODO: Fix this
+        );
+
+        const div = createElement(createElementData, document) as HTMLDivElement;
+
+        if (isHorizontal) {
+            // tableRect.left/right is used because the Inserter is always intended to be on the side
+            div.style.left = `${
+                isRTL
+                    ? tableRect.right
+                    : tableRect.left - (INSERTER_SIDE_LENGTH - 1 + 2 * INSERTER_BORDER_SIZE)
+            }px`;
+            div.style.top = `${tdRect.bottom - 8}px`;
+            (div.firstChild as HTMLElement).style.width = `${tableRect.right - tableRect.left}px`;
+        } else {
+            div.style.left = `${isRTL ? tdRect.left - 8 : tdRect.right - 8}px`;
+            // tableRect.top is used because the Inserter is always intended to be on top
+            div.style.top = `${
+                tableRect.top - (INSERTER_SIDE_LENGTH - 1 + 2 * INSERTER_BORDER_SIZE)
+            }px`;
+            (div.firstChild as HTMLElement).style.height = `${tableRect.bottom - tableRect.top}px`;
+        }
+
+        (anchorContainer || document.body).appendChild(div);
+
+        const handler = new TableInsertHandler(
+            div,
+            td,
+            table,
+            isHorizontal,
+            editor,
+            onInsert,
+            getOnMouseOut
+        );
+
+        return { div, featureHandler: handler, node: td };
+    }
+
+    return null;
+}
+
+class TableInsertHandler implements Disposable {
+    private onMouseOutEvent: null | ((ev: MouseEvent) => void);
+    constructor(
+        private div: HTMLDivElement,
+        private td: HTMLTableCellElement,
+        private table: HTMLTableElement,
+        private isHorizontal: boolean,
+        private editor: IStandaloneEditor,
+        private onInsert: () => void,
+        getOnMouseOut: (feature: HTMLElement) => (ev: MouseEvent) => void
+    ) {
+        this.div.addEventListener('click', this.insertTd);
+        this.onMouseOutEvent = getOnMouseOut(div);
+        this.div.addEventListener('mouseout', this.onMouseOutEvent);
+    }
+
+    dispose() {
+        this.div.removeEventListener('click', this.insertTd);
+
+        if (this.onMouseOutEvent) {
+            this.div.removeEventListener('mouseout', this.onMouseOutEvent);
+        }
+
+        this.onMouseOutEvent = null;
+    }
+
+    private insertTd = () => {
+        // Get cell coordinates
+        const columnIndex = this.td.cellIndex;
+        const row =
+            this.td.parentElement instanceof HTMLTableRowElement
```

Comment:
same here, use isElementOfType(). "instanceof" cannot work correctly in projection popout.